org-formation/organization.yaml

```diff
@@ -230,6 +231,15 @@ Organization:
       Tags:
         <<: !Include ./_default_org_tags.yaml
 
+  BMGFAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-bmgf
```

Comment:
@brian-bot should this account be called `bmgf` or should it be more specific, like some sort of project name?

Reply:
Done